packages/ui/src/app/pages/Profile/components/BannerSection.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { CloseButton } from '@/common/components/buttons'
+import { Arrow, QuestionIcon } from '@/common/components/icons'
+import { RowGapBlock } from '@/common/components/page/PageContent'
+import { DefaultTooltip } from '@/common/components/Tooltip'
+import { TextInlineMedium, TextMedium } from '@/common/components/typography/Text'
+import { Colors } from '@/common/constants'
+
+interface Props {
+  setShouldDismissBanner: (bool: boolean) => void
+}
+
+export const BannerSection = ({ setShouldDismissBanner }: Props) => {
+  return (
+    <Banner gap={8}>
+      <BannerHeader>
+        <BannerTitle>
+          <BannerTooltip>
+            <QuestionIcon />
+          </BannerTooltip>
+          <TextInlineMedium bold={true}>What is Joy Token?</TextInlineMedium>
+        </BannerTitle>
+        <CloseButton onClick={() => setShouldDismissBanner(true)} />
+      </BannerHeader>
+      <TextMedium>
+        JOY token is a native crypto asset of Joystream blockchain. It is used for platform governance, purchasing NFTs,
+        trading creator tokens, and covering blockchain processing fees. They are listed on{' '}
+        <a href="https://www.mexc.com/exchange/JOYSTREAM_USDT?_from=market" target="_blank">
+          MEXC exchange{' '}
+        </a>
+        under "JOYSTREAM" ticker.
+      </TextMedium>
+      <TextLink href="https://www.joystream.org/token/" target="_blank">
```

Comment:
This link should point to the EARN section too:
```suggestion
      <TextLink href="https://www.joystream.org/token#earn" target="_blank">
```